locales/en-US/server.ftl

```diff
@@ -81,6 +81,8 @@ notFoundPageDescription = Page not found.
 
 [[ Shot page ]]
 
+# This is the page title, put into the page <title></title>
```

Comment:
For example 

```
$ compare-locales l10n.toml . en-US
en-US/server.ftl
    ERROR: Unparsed content "# This is the page title, put into the page <title></title>" from line 84 column 1 to line 84 column 60
    WARNING: Parser error in en-US
```

That's because comments start with //